avaframe/in3Utils/geoTrans.py

```diff
@@ -701,6 +706,80 @@ def isCounterClockWise(path):
     return isCounterClock
 
 
+def getCellsAlongLine(header, lineDict, addBuffer=True):
```

Comment:
general question, is it an option to make a buffered line and look for the affected cells, maybe shapely offers tools for this, similar to the create ReleaseBuffer function..